src/Sentry.Unity/UnityViewHierarchyAttachmentContent.cs

```diff
@@ -72,14 +72,22 @@ internal void CreateNode(int remainingDepth, int maxChildCount, ViewHierarchyNod
     {
         var components = new List<Component>();
         transform.GetComponents(components);
+        var extras = new List<string>(components.Count);
+        foreach (var component in components)
+        {
+            if (component?.GetType().Name is { } componentName)
```

Comment:
The stack trace on https://github.com/getsentry/sentry-unity/issues/1918 led me to believe `GetComponents` is adding `null` to entries to the List it receives as argument.